custom_components/hacs/http.py

```diff
@@ -214,7 +214,7 @@ class Repository(HacsWebResponse):
             repository.status.new = False
             self.data.write()
 
-        render = self.render("repository", repository=repository, message=message)
+        render = self.render("repository", "repository", repository=repository, message=message)

```

Comment:
You open the repository view from all 3 tabs, when in a repository view there should not be any active tab (unless you add logic so it remembers it revert this change